src/log_database_proxy_model.cpp

```diff
@@ -42,9 +42,16 @@
 #include <QTimer>
 #include <QSettings>
 #include <swri_console/settings_keys.h>
+#include <QtGlobal>
+#include <algorithm>
+#include <iterator>
+
 
 namespace swri_console
 {
+QString failedSearchText="";//stores last failed search text, used to minimize looping through full data set, VCM 26 April 2017
+int failedSearchIndex = 0;//stores last index of failed search text, VCM 26 April 2017
```

Comment:
Do these really need to be global in the `swri_console` namespace, or could they be moved into the `LogDatabaseProxyModel` class?